Steepfile

```diff
@@ -31,13 +38,18 @@ target :lib do
   # library "pathname", "set"       # Standard libraries
   library 'logger', 'monitor'
   # library "strong_json"           # Gems
+  #
+  #   # configure_code_diagnostics(D::Ruby.strict)       # `strict` diagnostics setting
+  #   # configure_code_diagnostics(D::Ruby.lenient)      # `lenient` diagnostics setting
+  #   # configure_code_diagnostics do |hash|             # You can setup everything yourself
+  #   #   hash[D::Ruby::NoMethod] = :information
+  #   # end
```

Comment:
This is generated as part of the latest version of steep. Figured I'd leave it in so I can figure out how to actually use it.